_pages/dev/components/carousel-component.md

```diff
@@ -55,3 +55,27 @@ When there are no slides to navigate, the buttons do not appear. When a button i
 
 The number of carousel items in each slide is calculated. The calculation is driven by the minimum width per item and the available space.
 The given item width (`itemWidth`) can be specified in pixels or percentages. The calculation for the number of items per slide uses this `itemWidth` and the host element `clientWidth`. This makes the carousel reusable in different layouts (such as a one-column grid or a two-column grid), and enables the carousel to be fully responsive at the same time.
+
+### Configuring Slide Actions
+
+You can configure the actions for a slide. For example, adding Add-to-Cart buttons directly within your Merchandising Carousel to make it more efficient for consumers to add certain recommended products to their cart.
```

Comment:
In inner components, the user can plug in any kind of CMS components, not only "actions".  Should we rephrase the description?

Reply:
I think "actions" makes more sense because there's not really other UI elements you would put in that area - maybe promotional badges or something like, but it's likelier going to be buttons.